packages/react-scripts/package.json

```diff
@@ -60,6 +60,7 @@
     "postcss-loader": "0.13.0",
     "promise": "7.1.1",
     "react-dev-utils": "^0.2.1",
+    "react-hot-loader": "^3.0.0-beta.6",
```

Comment:
yeah, if it's working well in practice, then for sure. I meant "production" as we would all be using it day to day for dev. i'm sure its fine, I just haven't tried it out.